domains/geetank.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "afk-png",
+        "email": "githubgalactic.10@gmail.com"
+    },
+    "record": {
```

records, with "S", the letter "s" without this ("), you get it?
```suggestion
    "records": {
```